mewarpx/scripts_ancillary/tinyprofiler_plotting_example.py

```diff
@@ -186,6 +185,9 @@ def plot_selection_of_times(functions_to_plot, exclusive, avg, title, filename,
 
         dataframes.append(filtered_gf.dataframe)
 
+    if len(dataframes) > 20:
+        print(f"WARNING: {len(dataframes)} are set to be plotted, this will cause some distinct functions or regions to have the same color!")
```

Comment:
Break on 80 characters. Also, this warning is appropriate iff the tab20 colormap is used, so this warning should go around line 150, not here.